packages/ra-ui-materialui/src/list/Datagrid.js

```diff
@@ -93,8 +93,8 @@ const useStyles = makeStyles(
  *     </Datagrid>
  * </ReferenceManyField>
  */
-function Datagrid({ classes: classesOverride, ...props }) {
-    const classes = useStyles({ classes: classesOverride });
+const Datagrid = props => {
+    const classes = useStyles(props);
```

Comment:
According to MUI documentation, I doubt that works (or even worked before)
because we never use given props in the `makeStyles(theme => ({...}))` few lines upper

https://material-ui.com/styles/basics/#adapting-based-on-props

IMO, `useStyles();` would do the same

Reply:
We're not adapting based on props, we're allowing the user to pass a `classes` props to override the classes we provide. It's undocumented in the material-ui doc, but it works.

Reply:
Ok.